tests/cephadm/test_config.py

```diff
@@ -0,0 +1,62 @@
+import traceback
+
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class CephadmOpsFailure(Exception):
+    pass
+
+
+def set_multiple_public_network(cls):
+    """
+    To set multiple public network to ceph.
+    Args:
+        cls: cephadm instance object
+
+    Returns:
+        False -> Failed to set multiple public network to ceph
+        True -> Test passed to set multiple public network to ceph
+    """
+    out, _ = cls.shell(args=["ceph", "config", "get", "mon", "public_network"])
+    ceph_pub_network = out.rstrip()
+    mask = ceph_pub_network.rsplit(sep="/")
+    # logic to create new public ip by incrementing mask count
+    new_mask = int(mask[-1]) + 1
+    new_pub_network = f"{mask[0]}/{str(new_mask)}"
+    multiple_network = f"{ceph_pub_network},{new_pub_network}"
```

Comment:
@sunilangadi2 can you please point me on how we are getting multiple networks?

I don't think `ceph config get mon public_network` will give us multiple networks.

Reply:
@sunilkumarn417 Yes from cephci side by deafult bootstrap sets one public network 
But initially, there is bug with `cephadm` which is not able to set multiple network using `ceph config set mon public_network` so to test that scenario  created one more public network by incrementing mask count which is not really public network IP ,
We are just validating cephadm CLI functionality not the whole network functionality.